lib/elixir/lib/map_set.ex

```diff
@@ -351,16 +351,16 @@ defmodule MapSet do
   defp order_by_size(map1, map2), do: {map1, map2}
 
   defimpl Enumerable do
-    def reduce(set, acc, fun), do: Enumerable.List.reduce(MapSet.to_list(set), acc, fun)
-    def member?(set, val),     do: {:ok, MapSet.member?(set, val)}
-    def count(set),            do: {:ok, MapSet.size(set)}
+    def reduce(map_set, acc, fun), do: Enumerable.List.reduce(MapSet.to_list(map_set), acc, fun)
+    def member?(map_set, val),     do: {:ok, MapSet.member?(map_set, val)}
+    def count(map_set),            do: {:ok, MapSet.size(map_set)}
```

Comment:
One last thing @asummers, can I take advantage of this PR and ask you to remove the alignment from these lines? Right now they're aligned on `do:` but we're trying to move away from that; since you're already touching these lines, it would be very nice if you could do it in these PR 😃